telemetry/telemetry_test.go

```diff
@@ -197,6 +205,14 @@ func saveCertKeyPair(certPath, keyPath string) error {
 	certBytes := cert.Certificate[0]
 	keyBytes := x509.MarshalPKCS1PrivateKey(cert.PrivateKey.(*rsa.PrivateKey))
 
+	// Create the parent certs directory
+
+	dirPath := filepath.Dir(certPath)
+
+	if err := os.MkdirAll(dirPath, 0755); err != nil {
```

Comment:
>MkdirAll [](http://example.com/codeflow?start=14&length=8)

Could you prevent create dir inside test? How about just create the dir in codebase?

Reply:
Sure, will change.